include/deal.II/base/mpi_consensus_algorithms.h

```diff
@@ -141,23 +144,69 @@ namespace Utilities
        * <a href="https://en.wikipedia.org/wiki/Consensus_algorithm">consensus
        * problem</a>.
        *
-       * Dynamic-sparse means in this context:
-       * - By the time this function is called, the other processes do
+       * The problem consensus algorithms are trying to solve is this: Let's
+       * say you have $P$ processes that work together via MPI. Each (or at
+       * least some) of these want to send information to some of the other
+       * processes, or request information from other processes. No process
+       * knows which other process wants to communicate with them. The challenge
+       * is to determine who needs to talk to whom and what information needs to
+       * be sent, and to come up with an algorithm that ensures that this
+       * communication happens.
+       *
+       * That this is not a trivial problem can be seen by an analogy of the
+       * postal service. There, some senders may request information from some
+       * other participants in the postal service. So they send a letter that
+       * requests the information, but the recipients do not know how many such
+       * letters they need to expect (or that they should expect any at all).
+       * They also do not know how long they need to keep checking their mailbox
+       * for incoming requests. They can be considered reliable, however: We
```

Comment:
```suggestion
       * for incoming requests, however, can be considered reliable: We
```
Currently, it sounds like that the requests are reliable.